UNRELEASED.md

```diff
@@ -1,2 +1,9 @@
 # Unreleased
 
+**New:**
+ - bpk-component-scrollable-calendar:
+   - implemented BpkScrollableCalendarDate, built on top of BpkCalendarDate
```

Comment:
Whilst the scrollable calendar remains "private", there is no need to add changelog / unreleased entries :). So I'd remove:
```
**New:**
 - bpk-component-scrollable-calendar:
   - implemented BpkScrollableCalendarDate, built on top of BpkCalendarDate
```